components/InfoDrawer.js

```diff
@@ -100,11 +102,22 @@ const InfoDrawer = ({
                     <Typography variant="bodyMini" lightened bold>
                       ASSIGNEE
                     </Typography>
-                    <Avatar
-                      size="mini"
-                      // TODO: can we get the assignee information for each task in the process serializer
-                      src={assignee && assignee.imageUrl}
-                    />
+                    <Stack spacing={2} direction="row">
+                      //TODO: map function to iterate through array of assignees
+                      <AvatarWrapper
+                        badgeContent={
+                          assignee && assignee.id === completedBy.id ? (
```

Comment:
If the assignee id matches the id of the user that completed the task, show the check badge

Reply:
I'm imagining the assignee might have a completed attribute, but I need to flush this out in my backend work first.